toolchain/check/handle_pattern_list.cpp

```diff
@@ -26,20 +32,32 @@ auto HandleParseNode(Context& context, Parse::ImplicitParamListId node_id)
   return true;
 }
 
-auto HandleParseNode(Context& context, Parse::TuplePatternStartId node_id)
+static auto HandleTuplePatternStart(Context& context, Parse::NodeId node_id)
```

Comment:
How come the `node_id` is not more strictly typed as `Parse::TuplePatternStartId` here?